support-frontend/assets/pages/weekly-subscription-checkout/helpers/__tests__/deliveryDays.ts

```diff
@@ -1,53 +1,61 @@
 // ----- Imports ----- //
```

Comment:
Could we rename file to `deliveryDays.test.ts`? As I spent a while assuming was that earlier!